sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -889,7 +889,7 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
         call read_netcdf_3d_para(ncid3d,im,jm,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
         spval,VarName,cfr(ista_2l,jsta_2l,1),lm)
       else
-        VarName='cldfra'
+        VarName='cldfra_bl'
```

Comment:
@SamuelTrahanNOAA It appears that reading "cldfra" is needed by EMC side. I would suggest adding changes in the subroutine read_netcdf_2d_para for reading "cldfra_bl" when "cldfra" is missing in model output, such as:
```
iret = nf90_inq_varid(ncid,trim(varname),varid)
if (iret == 0 .and. trim(varname) == "cldfra") then
   iret = nf90_inq_varid(ncid,"cldfra_bl",varia) !!reading cldfra_bl" for cloud fraction
endif
...
...

```
Please let me know your thoughts.

Reply:
@ericaligo-NOAA, @ChunxiZhang-NOAA Could you answer Ming's question above?

Reply:
Isn't that determined by what you put in your diag table?

Reply:
I added logic to automatically choose cldfra or cldfra_bl. Could everyone please try the code and make sure it works for your configuration?

It'll use whichever variable is present. If both are present, it'll give a warning and use cldfra.

Reply:
@SamuelTrahanNOAA Your solution looks great to me.  I will test with RRFS model history files including cldfra. Thanks!

Reply:
@ericaligo-NOAA @hu5970 Could one of you please test this in the RRFS workflow?

Reply:
@SamuelTrahanNOAA I will test it and make sure it works!  Thanks.

Reply:
@SamuelTrahanNOAA it works fine for RRFS!  Thanks for the quick coding work.